content/md/assets/family-by-family-pam-migration.md

```diff
@@ -21,8 +21,8 @@ Other steps are automated, you just need to launch a few commands and your PAM a
 
 # When do you advise to use it?
 We advise you this migration if:
-- Your assets have different types (for example, you manage images, PDF files, videos).
-- AND each asset collection contains only a type of assets (for example, images or PDF files) because each asset collection will be then linked to a single asset family.
+- Your assets have **different types** (for example, you manage images, PDF files, videos).
+- AND **each asset collection contains only a type of assets** (for example, images or PDF files) because each asset collection will be then linked to a single asset family.
```

Comment:
```suggestion
- AND **each asset collection contains only one single type of asset** (for example, images or PDF files) because each asset collection will be then linked to a single asset family.
```